<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntregaActualizacion</class>
 <widget class="QDialog" name="EntregaActualizacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>335</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Entrega</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
	background-color: rgb(255, 255, 255);
}

QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 15px;
letter-spacing: 0.02em;
color: #6B7280;}



/* COMBOBOX */

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}



QDateEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
padding:8px;
border-radius:8px; }

QDateEdit::up-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::down-button {
min-width: 25px; min-height: 12px; background-color: #F3F4F6; 
border-radius: 6px;}

QDateEdit::up-arrow{
	image: url(:/menu/contraerarriba.png);
     width: 11px;
     height: 11px;
}

QDateEdit::down-arrow{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;}

QDateEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


/*  BOTON */

QPushButton{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
color:white;
padding:8px;
background: #3A4F50;
border-radius: 4px;}

QPushButton:hover{background: #344647;
border-radius: 4px; }

QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}

</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>4455</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}

QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 16px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titulo_accion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 14pt &quot;Roboto Black&quot;;</string>
    </property>
    <property name="text">
     <string>Edicion entrega informe</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_tutor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>32</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nueva Afiliación</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_closeadmi_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Cerrar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_closeadmi_3{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_closeadmi_3:hover{background:#ce4d3a ;
border-radius:11px; }


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image/source.qrc">
      <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-color:white;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
}


</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilo Base del Botón de la Paginación */
QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 10px 20px; /* Incrementamos el padding para un mejor tamaño de botón */
    background: #3f5758;
    border: none; /* Eliminamos el borde por defecto */
    border-radius: 8px; /* Ajustamos el border-radius */
}

/* Estilo del Botón de la Paginación al Pasar el Ratón (Hover) */
QPushButton:hover {
    background: #4a6b6c; /* Color de fondo más claro */
    border-radius: 8px;
}

/* Estilo del Botón de la Paginación al Hacer Clic (Pressed) */
QPushButton:pressed {
    background: #2e4546; /* Color de fondo más oscuro */
    border-radius: 8px;
}

#stackedWidget{background-color:white;
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;
}


</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">#page_3{
background-color: rgb(255, 255, 255);
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;

}</string>
     </property>
     <widget class="QRadioButton" name="rbt_no_memorandum">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NO</string>
      </property>
     </widget>
     <widget class="QLabel" name="pregunta_memorandum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Estado de entrega</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbt_si_memorandum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SI</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_registrar_informe">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>191</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>567</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Actualizar entrega</string>
      </property>
      <property name="icon">
       <iconset resource="../image/source.qrc">
        <normaloff>:/menu/ok.png</normaloff>:/menu/ok.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>21</height>
       </size>
      </property>
     </widget>
     <widget class="QDateEdit" name="fecha_informe">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>151</width>
        <height>40</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl9_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Fecha  entrega</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
     <widget class="QComboBox" name="cbo_meses_entrega">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxVisibleItems">
       <number>67</number>
      </property>
      <item>
       <property name="text">
        <string>Seleccione mes</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lbl9_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Meses entregados</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
